ivadomed/loader/utils.py

```diff
@@ -501,3 +503,81 @@ def reorient_image(arr, slice_axis, nib_ref, nib_ref_canonical):
     trans_orient = nib.orientations.ornt_transform(ras_orientation, ref_orientation)
     # apply transformation
     return nib.orientations.apply_orientation(arr_ras, trans_orient)
+
+def create_bids_dataframe(loader_params, derivatives):
+    """Create a dataframe containing all BIDS image files in a bids_path and their metadata.
+
+    Args:
+        loader_params (dict): Loader parameters, see :doc:`configuration_file` for more details.
+        derivatives (bool): If True, derivatives are indexed.
+
+    Returns:
+        df (pd.DataFrame): Dataframe containing all BIDS image files indexed and their metadata.
+    """
+
+    # Get bids_path, bids_config, target_suffix and extensions from loader parameters
+    bids_path = loader_params['bids_path']
+    bids_config = loader_params['bids_config']
```

Comment:
We should make this parameter optional in the config file (we could simply put None by default) and if it is None, we take the default file present in ivadomed/ivadomed/config, because in most cases we don't need a custom bids file.

Reply:
Fixed in 4e5eaf7
Let me know if this is what you had in mind.